mempool/src/proto/mod.rs

```diff
@@ -7,7 +7,41 @@
 use crate::proto::shared::*;
 use ::types::proto::*;
 
-pub mod mempool;
-pub mod mempool_client;
-pub mod mempool_grpc;
+pub mod mempool {
+    include!(concat!(env!("OUT_DIR"), "/mempool.rs"));
```

Comment:
where will this OUT_DIR be? can't we generate the proto files here?

Reply:
I commented on the other PR about this.